SU-HSDO/suhumsci
# SHS-5886: Remove hs_row module

# READY FOR REVIEW

## Summary
- Remove `hs_row` module files

## Need Review By (Date)
11/06

## Urgency
medium

## Steps to Test
1. Log in to the [Tugboat test site](https://pr1676-bis5onljufdfqsujlsnzadevflq7wwhx.tugboatqa.com/user/login)
Visit the [modules admin page](https://pr1676-bis5onljufdfqsujlsnzadevflq7wwhx.tugboatqa.com/admin/modules), search for "row". Confirm that there aren't any modules that match the filter.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208672271209144
